src/lib/Components/Bidding/Utils/findCountryNameByCountryCode.ts

```diff
@@ -0,0 +1,14 @@
+import { COUNTRY_SELECT_OPTIONS } from "lib/Components/CountrySelect"
+
+// finds a country's long name by the passed country code
+const findCountryNameByCountryCode = (countryCode: string): string | null => {
```

Comment:
```suggestion
export const findCountryNameByCountryCode = (countryCode: string): string | null => {
```

we can directly export here.